### DISCUSSION TITLE
Access Markdown source code of protected Wiki

### DISCUSSION THREAD

**Discussion by jcubic** [community/community#123632] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is there a way to get access to GitHub Wiki Markdown source code for non contributors when Wiki is protected?

I want to include documentation how to use ChatGPT and copy/paste Reference manual, so you can ask questions about my project. It would be nice if you can use markdown instead of text.

**Comment by nikhilsbuilds** [comment]
I understand your concern!

When a GitHub Wiki is protected, it means that only authorized users can access and edit the content. However, there are a few workarounds to access the Markdown source code without being a contributor:

1. **Use the GitHub API**: You can use the GitHub API to fetch the Wiki content programmatically. You'll need to create a GitHub Personal Access Token with the `repo` scope and use the `GET /repos/{owner}/{repo}/wikis/{wiki}/content/{path}` endpoint to retrieve the Wiki content. You can then parse the Markdown content as needed.
2. **Use a GitHub Wiki scraper**: There are several open-source projects that can scrape GitHub Wiki content, such as `github-wiki-scraper` or `wiki-scraper`. These tools can help you extract the Markdown content without requiring contributor access.
3. **Contact the Wiki owner**: If you have a legitimate reason for accessing the Wiki content, you can try contacting the Wiki owner or the repository maintainer to request access. They might be willing to share the content or provide alternative access methods.

Regarding your specific use case, you can use the Markdown source code to create a reference manual for ChatGPT. You can also consider creating a separate Markdown file for your project's documentation and referencing the ChatGPT manual as needed.

Remember to respect the terms of service and licensing agreements when using or sharing content from other sources.
